datadog/dogstatsd/base.py

```diff
@@ -50,7 +50,8 @@ class DogStatsd(object):
     def __init__(self, host=DEFAULT_HOST, port=DEFAULT_PORT, max_buffer_size=50, namespace=None,
                  constant_tags=None, use_ms=False, use_default_route=False,
                  socket_path=None, default_sample_rate=1, disable_telemetry=False,
-                 telemetry_min_flush_interval=DEFAULT_TELEMETRY_MIN_FLUSH_INTERVAL):
+                 telemetry_min_flush_interval=DEFAULT_TELEMETRY_MIN_FLUSH_INTERVAL,
+                 telemetry_host=None, telemetry_port=DEFAULT_PORT, telemetry_socket_path=None):
```

Comment:
Can we document the added params in the docstring please ?

Reply:
Absolutely, my bad.

Reply:
We should use None too for the port, for the case where users change the port and want the telemetry to follow.